inc/api/namespace.php

```diff
@@ -626,6 +626,14 @@ function get_top_data( $start, $end, ?Filter $filter = null ) {
 	$query = new WP_Query( $query_args );
 
 	foreach ( $query->posts as $i => $post ) {
+		$thumbnail_id = 0;
+		if ( $post->post_type === 'attachment' ) {
+			$thumbnail_id = $post->ID;
+		} else {
+			$thumbnail_id = get_post_thumbnail_id( $post ) ?: 0;
+		}
+		$thumbnail = $thumbnail_id ? wp_get_attachment_image_url( $thumbnail_id, '100x50' ) : '';
```

Comment:
Where is this image size coming from? I think we may be best off using the default `thumbnail` or `post-thumbnail` image size if it exists.